docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -1,47 +1,51 @@
+Working in the cloud has its advantages. However, unlike working on a desktop where you save your files to a local machine, you must either deploy your changes to your org, or commit them to source control to save your work somewhere permanent. Work saved in your Code Builder environment will be there when you come back but Code Builder is not suited for permanent-long-term storage. We recommended saving your work before you close the Code Builder tab.
+
+## Create Your Code Builder Environment
 
+1. From the App Launcher, find and open **Code Builder Dashboard**.
 
-## Let’s Get Started
-To get started you’ll do the following:
-1. Create a Code Builder environment.
-2. Create a Salesforce DX project.
-3. Connect to your org by logging in.
-Now you’re ready to use Code Builder.
+**Note:** If you don't see Code Builder Dashboard as an option, contact your admin to make sure you have the correct license and permission set assigned to you.
 
-## Connect to Your Dev Environment
-1. From the App Launcher, find and open **Code Builder**.
+2. Click **Launch** to launch your Code Builder environment. A new environment is created for you the first time to launch. We've also created an empty Salesforce DX project to help you get started.
 
-**Note:** If you don't see Code Builder as an option, or you see an empty screen when you launch it, contact your admin to make sure you have the correct license and permission set assigned to you.
+![Code Builder Dashboard](../../../images/cb_dashboard.png)
 
-2. Click **Get Started**.
-3. Click **New Project** to create a new Salesforce DX project.
-4. Give your project a name and choose **Standard** for project type.
+**Tip**: Once you have created your Code Builder environment, launch it from the dashboard at any time, or bookmark it for fast access.
 
-Next, connect an org to your Code Builder environment. You can choose to connect to a Salesforce org or a sandbox org.
+## Connect to a Salesforce Org
 
-1. Log in to your development environment, click **Allow** to grant access, then click **Next**.
-2. Choose an alias (nickname) for your org. If you work on many orgs, make sure the nickname helps you identify the org quickly in the future. 
-Your new Code Builder environment is now available on your dashboard.
-3. Click **Launch** to launch Code Builder in a new tab. 
-4. Now sit back and relax for a few minutes while Code Builder creates and configures your workspace. Code Builder might take a fews minutes to start up when it initially sets up.
+Next, connect an org to your Code Builder environment. During the course of development, you'll use different types of orgs for different stages. For example, it's common to use a Developer sandbox or Development Edition org during the development phase, and move to other sandbox types for integration, testing, and staging. Eventually, you'll deploy your changes to a production org. You can connect Code Builder to any of these orgs to deploy or retrieve metadata.
```

Comment:
```suggestion
Next, connect an org to your Code Builder environment. During the course of development, you'll use different types of orgs for different stages. For example, it's common to use a Developer sandbox or Developer Edition org during the development phase, and move to other sandbox types for integration, testing, and staging. Eventually, you'll deploy your changes to a production org. You can connect Code Builder to any of these orgs to deploy or retrieve metadata.
```

Reply:
If you borrowed that verbiage from DOCe help, I'll need to go find that typo. <lol>